docs/content/preview/deploy/manual-deployment/system-config.md

```diff
@@ -13,29 +13,84 @@ type: docs
 
 Perform the following configuration on each node in the cluster:
 
-- ntp or chrony
-- ulimits
-- transparent hugepages
+- set up time synchronization
+- set ulimits
+- enable transparent hugepages
 
 Keep in mind that, although YugabyteDB is PostgreSQL compatible and runs a postgres process, it is not a PostgreSQL distribution. The PostgreSQL it runs doesn't need the same OS and system resources that open source PostgreSQL requires. For this reason, the kernel configuration requirements are different.
 
 In particular, the main YugabyteDB process, the YB-TServer, is multi-threaded. As a result, you don't need to modify settings for shared memory and inter-process communication (IPC), because there is no inter-process communication or shared memory in a multi-threaded process model (all memory is shared by the same process).
 
-## ntp
+## Set up time synchronization
 
-If your instance does not have public Internet access, make sure the ntp package is installed:
+YugabyteDB relies on clock synchronization to guarantee consistency in distributed transactions. Chrony is the prefered NTP implementation for clock synchronization.
+
+### Install Chrony
+
+To install chrony, run:
 
 ```sh
-$ sudo yum install -y ntp
+$ sudo yum install -y chrony
 ```
 
-As of CentOS 8, `ntp` is no longer available and has been replaced by `chrony`. To install, run:
+### Configure Precision Time Protocol
+
+{{<tags/feature/tp>}} Precision Time Protocol (PTP) is a network protocol designed for highly accurate time synchronization across devices in a network. PTP provides microsecond-level accuracy. PTP relies on a PTP Hardware Clock (PHC), a dedicated physical clock device that enhances time synchronization accuracy.
+
+Currently, PTP is only available for AWS. To check if your AWS instance supports PTP and PHC, see [AWS PTP Hardware Clock](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/configure-ec2-ntp.html#connect-to-the-ptp-hardware-clock).
+
+Configure PTP using the `configure_ptp.sh` script in the bin directory of your YugabyteDB home directory as follows:
 
 ```sh
-$ sudo yum install -y chrony
+sudo bash ./bin/configure_ptp.sh
+```
+
+### Configure ClockBound
+
+{{<tags/feature/tp>}} [ClockBound](https://github.com/aws/clock-bound) is an open source daemon that allows you to compare timestamps to determine order for events and transactions, independent from the instances’ respective geographic locations. ClockBound provides a strict interval in which the reference time (true time) does exist.
```

Comment:
```suggestion
{{<tags/feature/tp>}} [ClockBound](https://github.com/aws/clock-bound) is an open source daemon that allows you to compare timestamps to determine order for events and transactions, independent of an instance's geographic location. ClockBound provides a strict interval in which the reference time (true time) exists.
```

Reply:
Done

Added a small change to read-restart-error.md as well.